jobboard/tests.py

```diff
@@ -182,3 +182,41 @@ def test_page_display_to_user_that_is_recruiter(client):
     jobs_to_display = Job.get_jobs_posted_on_or_after_specific_date(date_six_months_ago)
     # test that jobs displayed are only jobs created 6 months ago
     assert response.context['jobs_to_display'] == jobs_to_display
+
+
+@pytest.mark.django_db
+def test_navbar_links_are_correct_for_student_user(client, example_student_a):
+    client.force_login(example_student_a.user)
```

Comment:
why `force_login`?

Reply:
@guy9050 why not? It's easier to use instead of typing in the username and password for the user and we used it in plenty of tests already